src/config/typeorm.config.ts

```diff
@@ -1,6 +1,8 @@
 import { TypeOrmModuleOptions } from '@nestjs/typeorm';
+
 import * as config from 'config';
 
+// tslint:disable-next-line: typedef
 const dbConfig = config.get('db');
```

Comment:
add type any and disable no-any

Reply:
i prefer if you create a type for it like you did in other places